knowledgebase/change-billing-email.md

```diff
@@ -0,0 +1,17 @@
+---
+date: 2023-09-08
+---
```

Comment:
To have the actual title (vs the filename) appear in the left handside KB list

```suggestion
title: How do I change my Billing Contact in ClickHouse Cloud?
---
```

Reply:
Just FYI: the "title" is not needed in KB articles if you're happy with the page heading (the # text)